packages/design-tokens/docs/components/TokensTable.tsx

```diff
@@ -10,26 +12,36 @@ import {
 } from "../../../components/src/components/Table/";
 import { Token } from "../types/Token";
 import { buildTokensTableRows } from "../utils";
+import { TokenRow } from "../types/TokenRow";
 import { TOKEN_COLUMNS } from "./constants";
 import { TokenName } from "./TokenName";
 
 type TokenTypes = keyof typeof TOKEN_COLUMNS;
 
 export const TokensTable = ({
-  type,
   data,
 }: {
   type: TokenTypes;
   data: Record<string, Record<string, Token>>;
 }): JSX.Element => {
-  const columns = TOKEN_COLUMNS[type];
-  const rows = buildTokensTableRows(columns, data);
+  const tokenNames: Array<string> = filter(
+    keys(data),
+    (item) => item !== "default",
+  );
+  const columnGroups = map(tokenNames, (token) => {
+    return TOKEN_COLUMNS[token];
+  });
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const rowGroups: TokenRow[][] = map(columnGroups, (columns: any, index) => {
+    const token = tokenNames[index];
```

Comment:
sorry, forgot to mention that this `any` can go away
```suggestion
  const rowGroups: TokenRow[][] = map(columnGroups, (columns, index) => {
    const token = tokenNames[index];
```